docs/extra-containers.md

```diff
@@ -0,0 +1,64 @@
+---
+order: 100
+---
+# Extra Containers
+
+Local Server provides a series of Docker containers for the base Altis behaviour. Modules and other packages may add additional containers (typically called "sidecar" containers) or alter the container behaviour through the extension system.
+
+
+## How Local Server works
+
+Local Server internally uses Docker Compose to create and manage containers for the Altis environment. When a user runs the `composer server start` command, Local Server dynamically provisions a `docker-compose.yml` file based on the user's preferences. (This file is only regenerated when starting Local Server to avoid conflicts or surprising behaviour for users.)
```

Comment:
```suggestion
Local Server internally uses Docker Compose to create and manages containers for the Altis environment. When a user runs the `composer server start` command, Local Server dynamically provisions a `docker-compose.yml` file based on the user's preferences. (This file is only regenerated when starting Local Server to avoid conflicts or surprising behaviour for users.)
```

Reply:
Could you also wrap lines at 132 chars please? As per our MD [linting standards](https://github.com/humanmade/altis-dev-tools-command/blob/db0985a71418a4e11b2b287b30827db4c6d42d0f/templates/docslint/markdownlint.yaml#L61).